cluster/down.sh

```diff
@@ -2,4 +2,11 @@
 
 source ./cluster/gocli.sh
 
+echo "Bringing down cluster and client ..."
 $gocli rm
+
+# clean up unused docker volumes
+if [ "$(docker volume ls | wc -l)" -gt 1 ]; then
```

Comment:
Should you check for only dangling=true so something like:
```bash
if [ "$(docker volume ls -qf dangling=true | wc -l)" -gt 0]; then
```

Reply:
These are isolated inside the test VM so I'm not sure we can't just treat this as our own env and completely clean it.  That being said, I like the suggested refactor from @jeffvance

Reply:
Could this interfere with other docker workflows happening on the same machine?  Should we try to find and remove only images that this infrastructure created?

Reply:
Given that this runs in the VM (Cluster) built by cluster-up seems like these are *all* our docker workflows.  I'd suggest either the refactor @jeffvance suggested, or even a slight variant and just drop the dangling=true.  If you reworked slightly with either of those I'd be +2/A on this.